src/daemon/jsonrpc/api.rs

```diff
@@ -333,6 +337,123 @@ impl RpcApi for RpcImpl {
         }))
     }
 
+    fn help(&self, meta: Self::Metadata) -> jsonrpc_core::Result<serde_json::Value> {
```

Comment:
You don't use `meta` here, yet you have to keep it in the function declaration. To indicate you don't use it (and quiet the compiler warning), please prefix the variable name with an underscore (or just use an underscore instead):
```diff
diff --git a/src/daemon/jsonrpc/api.rs b/src/daemon/jsonrpc/api.rs
index 76a4c79..e11a11d 100644
--- a/src/daemon/jsonrpc/api.rs
+++ b/src/daemon/jsonrpc/api.rs
@@ -337,7 +337,7 @@ impl RpcApi for RpcImpl {
         }))
     }
 
-    fn help(&self, meta: Self::Metadata) -> jsonrpc_core::Result<serde_json::Value> {
+    fn help(&self, _: Self::Metadata) -> jsonrpc_core::Result<serde_json::Value> {
         Ok(json!(
         {
             "commands":[
```